### ISSUE TITLE
cutoff_radius limit not necessary in structured (periodic) porous media?

### ISSUE THREAD

**Issue by RolfStierle** [feos-org/feos#202]
https://github.com/feos-org/feos/blob/01d82386b4f34daf4a607fe8aaab637b6d7cce22/feos-dft/src/adsorption/pore3d.rs#L140

For periodic porous media, wouldn't we want to include the molecular interactions of the periodic images, since the porous media is also periodic? This would make this check obsolete.

**Comment by prehner** [comment]
Yes, but currently FeOs only determines the closest periodic image for every solid atom rather than every periodic image within the cutoff. When this is fixed, the check becomes obsolete.
